### PR TITLE
Changes to Community section

### PR DESCRIPTION
**Pull request by mjbrender**
Most importantly, highlighting projects that are active, awesome and should be used! :trophy: 

Secondarily, removal of outdated images that are bettered suited for our company [catalogue](http://basho.com/design-assets-hub/).

### CODE REVIEW

**Comment by mjbrender** [comment]
The last commit adds links to our latest OSS projects for Mesos and Riak Explorer. Please merge as you were planning to @laurenrother. And FYI @tylerhannan @drewkerrigan @sargun :hand:

File: data/downloads_gen.yml [hunk 1]
```diff
@@ -4822,6 +4822,212 @@ riak:
               file: riak-2.0.5.tar.gz
               static_file: http://s3.amazonaws.com/downloads.basho.com/riak/2.0/2.0.5/riak-2.0.5.tar.gz
               file_size: 17805547
+  2.1.1:
```
Comment (laurenrother) [inline data/downloads_gen.yml]: What's all this?

**Comment by laurenrother** [comment]
@mjbrender Looks good! I left a comment on one file whose edits I don't quite understand. Will merge once we've chatted about it.

File: data/downloads_gen.yml (hunk 1 above)
Comment (mjbrender) [inline data/downloads_gen.yml]: Ah crap. I thought that was an artifact of the platform that you'd understand @laurenrother. Working on comparing to upstream and removing what shouldn't bet there.

**Comment by mjbrender** [comment]
@laurenrother much better. Let me know if you think so too :shipit:

**Comment by laurenrother** [comment]
:cake:

**Comment by mjbrender** [comment]
Good catch @dmitrizagidulin. Opening a quick PR for it.

**Pull request merged**